projects/tuist/fixtures/visionos_app_with_remote_swift_package/Project.swift

```diff
@@ -0,0 +1,37 @@
+import ProjectDescription
+
+let project = Project(
+    name: "App",
+    packages: [
+        .package(url: "https://github.com/ReactiveX/RxSwift", .upToNextMajor(from: "5.0.0")),
```

Comment:
Why are we adding this to this fixture? Is this relevant for the functionality?
- If not, I would remove it
- If yes, I would use Dependencies.swift instead

Reply:
I copied from ios_app_with_remote_swift_package because with only one fixture it seemed this most likely to not work... I figured if nothing else this tests that a dependency without support for `visionOS` explicitly declared
I can remove it though

Reply:
Removed. Should be resolved!